spec/integration/plos_details_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'spec_helper.rb'
+
+describe PlosDetails, :vcr do
+  let(:plos_paper) do
+    create(:plos_paper, id: 1, reader: 0, doi: '10.1371/journal.pone.0089204')
+  end
+  let(:job) {PlosDetails.new}
```

Comment:
You don't need this line if you use `subject` instead of `job` in the test. As a side-note: You can override the default `subject` by calling it with a block on the same level as the `let`:

```ruby
subject { described_class.new(:foo, :bar) }
```